marketplace-ui/src/app/modules/product/product-detail/product-detail.component.ts

```diff
@@ -281,4 +285,10 @@ export class ProductDetailComponent {
     return this.detailTabsForDropdown.filter(tab => this.getContent(tab.value));
   }
 
+  convertTagToVersion(tag: string) : string {
```

Comment:
I remember I prepare this kind of method somewhere else, let check it again

Reply:
You prepared it in the BE. This is in the FE